docs/_includes/Statements.md

```diff
@@ -743,17 +743,66 @@ Because the expect expression and optional second argument are compiled, they ca
 `assume` for the verifier, but also inserts a run-time check that the 
 assumption is indeed correct (for the test cases used at run-time).
 
-Paired `assert` and `expect` statements 
-(with `assert` before `expect`) checking the
-same expression can be used to do runtime checking before there is a successful proof
-of an assert statement (or to help debug one that is unprovable).[^expect]
+Here are a few use-cases for the `expect` statement.
 
-[^expect]: Aside from difficulties in constructing a successful proof, 
-paired (consecutive) `assert` and `expect` statements in a program should always produce the 
-same results, except if the compiler is faulty. Of course, the `expect` statement only checks the 
-test cases for which the program is run.
+A) To check the specifications of external methods.
 
-Using `...` as the argument of the statement is part of module refinement, as described [here](#sec-module-refinement).
+Consider an external method `Random` that takes a `nat` as input and returns
+a `nat` value that is less than the input. Such a method could be
+specified as
+```dafny
+method {:extern} Random(n: nat) returns (r: nat)
+  ensures r < n
+```
+But because there is no body for `Random` (only the external non-dafny implementation), it cannot be verified that `Random` actually satisfies this specification.
```

Comment:
```suggestion
But because there is no body for `Random` (only the external non-dafny implementation),
it cannot be verified that `Random` actually satisfies this specification.
```